python/ray/includes/gcs_client.pxi

```diff
@@ -530,7 +530,7 @@ cdef class InnerGcsClient:
                 .GetClusterResourceState(timeout_ms, serialized_reply)
             )
 
-        return serialized_reply
+        return serialized_reply 
```

Comment:
It seems that there is a extra blank at the end of the line, It's better to run format.sh script in advance.